infrastructure/installer_header.sh

```diff
@@ -51,17 +51,21 @@ fi
 if [[ -e ~/.zshrc ]]
 then
     echo "export SPRYKER_SDK_PATH=\"$DESTINATION\"" >> ~/.zshrc && \
-    echo "alias spryker-sdk=\$SPRYKER_SDK_PATH\"/bin/spryker-sdk.sh\"" >> ~/.zshrc
-    echo 'Created alias in ~/.zshrc'
+    echo "alias spryker-sdk=\$SPRYKER_SDK_PATH\"/bin/spryker-sdk.sh\"" >> ~/.zshrc && \
```

Comment:
Currently, we have an issue with aliases creation, because our installer can create as many aliases as it was executed. If  `spryker-sdk` alias exists installer should rewrite it instead of making a new one. 

 My suggestion is to make a couple of functions for checking alias, alias creation and alias replacement(hint: `sed` command and improve existing logic using new functions.

Reply:
+